### PR TITLE
mes-MOTStatus

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Added support for checking MOT into des
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
imageimage
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by andrewsetterfield — APPROVED** [review]

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]
add this to remove my approval

**Review by andrewsetterfield — APPROVED** [review]

File: src/app/pages/waiting-room-to-car/components/mot-components/alternate-mot-evidence/alternate-mot-evidence.component.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  @Output()
+  alternateEvidenceChange = new EventEmitter<boolean>();
+
+  ngOnInit() {
+    this.setupControl();
+  }
+
+  setupControl(): void {
+    if (!this.formControl) {
+      this.formControl = null;
```
Comment (gittins7) [inline src/app/pages/waiting-room-to-car/components/mot-components/alternate-mot-evidence/alternate-mot-evidence.component.ts]: is there a need to set this to null before making it an untypedFormControl?

File: src/app/pages/waiting-room-to-car/components/vehicle-registration/__tests__/vehicle-registration.spec.ts [hunk 2]
```diff
@@ -16,9 +19,11 @@ describe('VehicleRegistrationComponent', () => {
   beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
       declarations: [VehicleRegistrationComponent],
+      providers: [Store],
       imports: [IonicModule, AppModule, ReactiveFormsModule],
     });
 
+    // store$ = TestBed.inject(Store);
```
Comment (gittins7) [inline src/app/pages/waiting-room-to-car/components/vehicle-registration/__tests__/vehicle-registration.spec.ts]: does this need to be re-introduced or can it be deleted?

File: src/app/shared/classes/test-flow-base-pages/office/office-base-page.ts [hunk 3]
```diff
@@ -631,4 +635,9 @@ export abstract class OfficeBasePageComponent extends PracticeableBasePageCompon
     });
     await this.finishTestModal.present();
   }
+
+  alternativeMOTEvidenceDescriptionUpdated(evidence: string) {
+    console.log(evidence);
```
Comment (gittins7) [inline src/app/shared/classes/test-flow-base-pages/office/office-base-page.ts]: stray console.log()

**Review by gittins7 — APPROVED** [review]

**Pull request merged**